package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "archiver-jsdoc-theme": "^1.1.3",
     "chai": "^4.2.0",
+    "chai-spies": "^1.0.0",
```

Comment:
[✓] I successfully ran the tests locally with this helper
[   ] It is up to the maintainer if they want to add a new library, or if they have other techniques for this type of test...